app/controllers/spree/admin/active_sales_controller.rb

```diff
@@ -49,13 +49,11 @@ def collection
           return @collection if @collection.present?
           params[:q] ||= {}
           params[:q][:deleted_at_null] ||= "1"
-          
-          params[:q][:s] ||= "name asc"
 
           @search = super.ransack(params[:q])
 
           @search = Spree::ActiveSale.includes(:active_sale_events).ransack(params[:q])
-          @collection = @search.result.page(params[:page]).per(Spree::ActiveSaleConfig[:admin_active_sales_per_page])
+          @collection = @search.result.page(params[:page]).per(Spree::ActiveSaleConfig[:admin_active_sales_per_page]).order(position: :asc)
```

Comment:
@ankitkalia1195 check if we need this even after adding a `default_scope` for the same thing.